definitions/npm/hoist-non-react-statics_v2.x.x/flow_v0.54.1-/hoist-non-react-statics_v2.x.x.js

```diff
@@ -0,0 +1,12 @@
+declare module 'hoist-non-react-statics' {
+  /*
+    S - source component statics
+    TP - target component props
+    SP - additional source component props
+  */
+  declare module.exports: <TP, SP, S>(
+    target: React$ComponentType<TP>,
+    source: React$ComponentType<TP & SP> & S,
+    blacklist?: { [key: $Keys<S>]: boolean }
+  ) => React$ComponentType<TP> & S; // TODO: handle blacklisted properties
```

Comment:
@marcind do you have an idea how the resulting statics keys here can be filtered according to blacklist type?